Makefile

```diff
@@ -75,6 +82,9 @@ ifeq ("${RP_IMAGE_ACR}-$(BRANCH)","arointsvc-master")
 		docker push arointsvc.azurecr.io/aro:latest
 endif
 
+publish-image-autorest: image-autorest
+	docker push ${AUTOREST_IMAGE}
```

Comment:
It seems there are only a select few who can do this, is a Makefile target necessary?

Reply:
I think that we should fix the privs for everyone. This just makes it easier to make updates in the future.

Reply:
its no difference than other images. Just the fact some people can't run it does not mean I need to craft command manually each time :)